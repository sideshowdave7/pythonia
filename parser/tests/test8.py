x = 3 if 1 == 2 else 4
y = 3 if False else 8 - 7/2

