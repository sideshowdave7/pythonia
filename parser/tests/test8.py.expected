(program
 (= (x) (if (comparison 1 ("==" 2)) 3 4))
 (= (y) (if False 3 (arith 8 ("-" (term 7 ("/" 2)))))))

